root/static/scripts/edit/URLCleanup.js

```diff
@@ -833,7 +833,11 @@ const CLEANUPS: CleanupEntries = {
     },
   },
   'bandcamp': {
-    match: [new RegExp('^(https?://)?([^/]+)\\.bandcamp\\.com', 'i')],
+    match: [new RegExp(
+      '^(https?://)?([^/]+)\\.bandcamp\\.com' +
+      '(?:[?#]+.*|/(?!campaign)([^?#]*)?.*)?$',
```

Comment:
Probably unintended, but this also blocks `https://foo.bandcamp.com/campaignfoo`.

Not sure I understand why the query/hash strings need to be matched there -- can't we just check that `/campaign/` doesn't follow the TLD?

```suggestion
      '^(https?://)?([^/]+)\\.bandcamp\\.com' +
      '(?!/campaign/)',
```

Reply:
Oh... I guess we could just do that, lol. dunno why I never thought of that and was obsessed with making it work against ?!campaign